src/config.ts

```diff
@@ -65,6 +65,12 @@ export type Config = {
     enabled: boolean;
     proxy: string;
   };
+  whitelist: {
+    enabled: boolean;
+    domain: string;
```

Comment:
```suggestion
    nip05Domain?: string;
    pubkeys?: string[];
```
I would add a manual list of pubkeys in case someone wants a short hard-coded list. and I would rename `domain` to something like `nip04Domian` to make it clear what its fetching